runtime/src/tests/staking.rs

```diff
@@ -28,52 +28,53 @@ fn active_era_transitions() {
-		let usable = Balances::usable_balance(&stash);
-		assert_eq!(0, usable);
-
-		// unbond, efective after BondingPeriod has passed
-		assert_ok!(Staking::unbond(RuntimeOrigin::signed(controller.clone()), value));
-		assert_eq!(value, Staking::total_stake(&controller).unwrap());
-		let usable = Balances::usable_balance(&stash);
-		assert_eq!(0, usable);
-
-		// Cant withdraw before minimum staking period has elapsed; NOOPs
-		assert_ok!(Staking::withdraw_unbonded(RuntimeOrigin::signed(controller.clone()), 0));
-		assert_eq!(value, Staking::total_stake(&controller).unwrap());
-		let usable = Balances::usable_balance(&stash);
-		assert_eq!(0, usable);
-
-		// unbonding defered until bonding eras pass
-		height += BLOCKS_UNTIL_UNBONDED;
-
-		Trivial::<Session, Runtime>::roll_to(height);
-
-		assert_ok!(Staking::withdraw_unbonded(RuntimeOrigin::signed(controller), 0));
-		let usable = Balances::usable_balance(&stash);
-		assert_eq!(value, usable);
-		assert!(Staking::total_stake(&stash).is_none());
-	});
+	// TODO
+	// let stash = generate_account("stash");
+	// let controller = generate_account("controller");
+	// let value = <Balances as Currency<AccountId>>::minimum_balance();
+	//
+	// ExtBuilder::<()>::default().build_sans_config().execute_with(|| {
+	// 	let mut height = 1;
+	// 	Trivial::<Session, Runtime>::roll_to(height);
+	//
+	// 	// prime balance to create the account
+	// 	let _ = Balances::deposit_creating(&stash, value);
+	// 	let usable = Balances::usable_balance(&stash);
+	// 	assert_eq!(value, usable);
+	//
+	// 	// lock stash's funds under a controller
+	// 	assert_ok!(Staking::bond(
+	// 		RuntimeOrigin::signed(stash.clone()),
+	// 		controller.clone().into(),
+	// 		value,
+	// 		RewardDestination::Stash,
+	// 	));
+	//
+	// 	// stake is found and usable balance dropped
+	// 	assert_eq!(value, Staking::total_stake(&controller).unwrap());
+	// 	let usable = Balances::usable_balance(&stash);
+	// 	assert_eq!(0, usable);
+	//
+	// 	// unbond, efective after BondingPeriod has passed
+	// 	assert_ok!(Staking::unbond(RuntimeOrigin::signed(controller.clone()), value));
+	// 	assert_eq!(value, Staking::total_stake(&controller).unwrap());
+	// 	let usable = Balances::usable_balance(&stash);
+	// 	assert_eq!(0, usable);
+	//
+	// 	// Cant withdraw before minimum staking period has elapsed; NOOPs
+	// 	assert_ok!(Staking::withdraw_unbonded(RuntimeOrigin::signed(controller.clone()), 0));
+	// 	assert_eq!(value, Staking::total_stake(&controller).unwrap());
+	// 	let usable = Balances::usable_balance(&stash);
+	// 	assert_eq!(0, usable);
+	//
+	// 	// unbonding defered until bonding eras pass
+	// 	height += BLOCKS_UNTIL_UNBONDED;
+	//
+	// 	Trivial::<Session, Runtime>::roll_to(height);
+	//
+	// 	assert_ok!(Staking::withdraw_unbonded(RuntimeOrigin::signed(controller), 0));
+	// 	let usable = Balances::usable_balance(&stash);
+	// 	assert_eq!(value, usable);
+	// 	assert!(Staking::total_stake(&stash).is_ok());
```

Comment:
@CAGS295 from what I can tell `total_stake()` has been removed, see https://github.com/paritytech/substrate/compare/polkadot-v0.9.32...polkadot-v0.9.38,  which is weird b/c the error that I get when the above test func is uncommented is 

```
error[E0599]: no method named `is_none` found for enum `Result` in the current scope
  --> runtime/src/tests/staking.rs:77:40
   |
77 |         assert!(Staking::total_stake(&stash).is_none());
   |                                              ^^^^^^^ help: there is a method with a similar name: `is_ok`
```

It doesn't tell me that there is no such function.  I wasn't able to find a definition for `total_stake` in the new `polkadot-v0.9.38` branch. Not sure how is that supposed to work.

Reply:
The implementation was moved to the `StakingInterface` trait.